modules/nf-core/goleft/indexcov/tests/main.nf.test

```diff
@@ -0,0 +1,114 @@
+                    ],
+                    [
+                        file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/bam/test.single_end.sorted.bam.bai", checkIfExists: true),
+                        file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam.bai", checkIfExists: true)                                                                                       
+                    ],
+                ])
+
+                input[1] = Channel.of(
+                    [
+                        [:],
+                        file(params.modules_testdata_base_path + "genomics/sarscov2/genome/genome.fasta.fai", checkIfExists: true)
+                    ]
+                )
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+
+    test("sarscov2 - crai") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    [
+                        file(params.modules_testdata_base_path + "genomics/homo_sapiens/illumina/cram/test.paired_end.sorted.cram", checkIfExists: true),
+                        file(params.modules_testdata_base_path + "genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram", checkIfExists: true)
+                    ],
+                    [
+                        file(params.modules_testdata_base_path + "genomics/homo_sapiens/illumina/cram/test.paired_end.markduplicates.sorted.cram.crai", checkIfExists: true),
+                        file(params.modules_testdata_base_path + "genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram.crai", checkIfExists: true)
+                    ]
+                ])
+
+                input[1] = Channel.of(
+                    [
+                        [:],
+                        file(params.modules_testdata_base_path + "genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai", checkIfExists: true)
+                    ]
+                )
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    [],
+                    []
+                ])
+
+                input[1] = Channel.of([
+                    [:],
+                    []
+                ])
```

Comment:
Mmm, this feels a bit weird, to be testing completely empty channels in the stub